### DISCUSSION TITLE
Can't save hidden file in windows

### DISCUSSION THREAD

**Discussion by Tyarel8** [helix-editor/helix#10045] [General] [answered]
### Summary

I can't save a hidden file when editing it in helix. If I go to the file explorer and unhide the file, it works correctly. I have also tried doing `sudo hx .stignore` using the [gsudo](https://github.com/gerardog/gsudo) program but that gave the same error.

### Reproduction Steps

 

I tried this:

1. `hx .stignore` (.stignore is a hidden file)
2. `:w` in helix

I expected this to happen:
The file to be saved.

Instead, this happened:
`Access is denied. (os error 5)`

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
2024-03-29T23:55:35.188 helix_view::editor [ERROR] editor error: Access is denied. (os error 5)
```
</details>

### Platform

Windows

### Terminal Emulator

wezterm 20240203-110809-5046fc22

### Installation Method

scoop

### Helix Version

helix 23.10 (f6021dd0)

**Comment by kirawi** [comment]
Are you able to use another editor to write that file? Have you also tried running in an administrator mode instead (which would point to gsudo as the issue)?

**Reply by Tyarel8** [reply]
I can edit the file with vscode and notepad without issue, and I have tried using helix in a administrator console with both cmd and pwsh with the same issue.

**Reply by Tyarel8** [reply]
I have now tried editing the file with neovim and that worked, but it unhidded the file.

**Reply by Tyarel8** [reply, marked as answer]
Just tried in `helix 24.3 (2cadec0b)` and it works fine, so case closed, thanks for the help 👍.
